scripts/measures/2018/enrich-measures-data.js

```diff
@@ -128,6 +130,29 @@ function mergeGeneratedEcqmData(measures) {
   });
 }
 
+/*
+ * A function that uses an identifier in `quality-strata.csv` to map a numerator uuid to a strata.
```

Comment:
Ah I see. The identifier is the uuid right? If so, it seems confusing to reference it separately. How about combining these two lines into:

```Uses numeratorUuid field as a common id to map each strata name (only in `enriched-measures-data-quality.json`) to a particular strata (in `quality-strata.csv`)```

Reply:
👍